package.json

```diff
@@ -19,6 +19,7 @@
   "files": [
     "eslint-plugin-prettier.d.ts",
     "eslint-plugin-prettier.js",
+    "recommended.js",
```

Comment:
Can you add a `recommended.d.ts` accordingly?

Reply:
Sure! How is `recommended.d.ts` used when it comes to an eslint plugin?

Reply:
https://github.com/eslint-types/eslint-define-config